ssn/chapters/Conformance.html

```diff
@@ -1,12 +1,9 @@
-<!-- <section id="conformance"> -->
-<!--    <h2>Conformance</h2> inserted automatically -->
 <p>Data conforms to SSN if:</p>
 <ul>
-  <li> Access to data is organized around <a href=https://www.w3.org/TR/vocab-ssn/#Actuations>Actuations</a>, <a href=https://www.w3.org/TR/vocab-ssn/#Observations>Observations</a> and <a href=https://www.w3.org/TR/vocab-ssn/#Sampling>Samplings</a>, which represent executions of Actuation, Observation and Sampling procedures, implemented by Actuators, Sensors and Samplers </li>
+  <li> Access to data is organized around Actuations, Observations and Samplings, which represent executions of Actuating, Observing and Sampling procedures, implemented by Actuators, Sensors and Samplers </li>
```

Comment:
Clarity is increased by the terminal comma in each phrase of this sentence. 
```suggestion
  <li> Access to data is organized around Actuations, Observations, and Samplings, which respectively represent executions of Actuating, Observing, and Sampling procedures, which are in turn respectively implemented by Actuators, Sensors, and Samplers </li>
```